src/main/resources/templates/email/ltft-rejected/v1.0.0.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+  <title>LTFT Rejected</title>
+</head>
+<body>
+<h1>Email Message</h1>
+<h2>Subject</h2>
+<th:block th:fragment="subject">
+  NHS England<th:block th:text="${not #strings.isEmpty(var?.programmeMembership?.managingDeanery)} ? | - ${var.programmeMembership.managingDeanery}| : _"></th:block> - Notification of Your Less Than Full Time (LTFT) Application Status - <th:block th:text="${not #strings.isEmpty(var?.formRef)} ? ${var.formRef} : 'unknown form reference'"></th:block>
+</th:block><h2>Content</h2>
+<th:block th:fragment="content" th:with="subjectRefs = |GMC: ${not #strings.isEmpty(var?.personalDetails?.gmcNumber) ? var.personalDetails.gmcNumber : 'unknown'}, LTFT Ref No: ${not #strings.isEmpty(var?.formRef) ? var.formRef : 'unknown'}|">
+  <div th:replace="fragments/retry/v1.0.0 :: retry(${originallySentOn})"></div>
+  <div style="text-align:right"><img style="margin-right:25px; height:100px" src='https://trainee.tis.nhs.uk/nhse-logo.png' alt="NHS England logo"/></div>
+  <p th:replace="~{fragments/greeting/v1.0.0 :: greeting(${familyName})}"></p>
+  <p>
+    Thank you for submitting your application to train at Less Than Full Time.
+  </p>
+  <p>
+    Unfortunately your application has been rejected for the following reasons:
+    <strong><th:block th:text="${not #strings.isEmpty(var?.stateDetail?.reason)} ? ${var.stateDetail.reason} : 'unknown'"></th:block></strong>
+  </p>
+  <p>
```

Comment:
From the screenshot, it looks like we are getting two blank lines between paragraphs - is that right? Might need to close the </p> on the same line as the paragraph text.